tax/set.go

```diff
@@ -133,7 +133,7 @@ func (c *Combo) prepare(r *Regime, zone l10n.Code, date cal.Date) error {
 // the rate field.
 func (c *Combo) UnmarshalJSON(data []byte) error {
 	type Alias Combo
-	aux := &struct {
```

Comment:
Same as before: this change should better be part of https://github.com/invopop/gobl/pull/234